docs/Shell-Completion.md

```diff
@@ -43,7 +43,12 @@ if type brew &>/dev/null; then
 fi
 ```
 
-This must be done before `compinit` is called. Note that if you are using Oh My Zsh, it will call `compinit` for you, so this must be done before you call `oh-my-zsh.sh`.
+This must be done before `compinit` is called. Note that if you are using Oh My Zsh, it will call `compinit` for you, so this must be done before you call `oh-my-zsh.sh`. This may be done by adding the following line to your `~/.zprofile` file, instead of modifying your `~/.zshrc` as above:
+
+```diff
+  eval $(/opt/homebrew/bin/brew shellenv)
++ export FPATH=$(brew --prefix)/share/zsh/site-functions:$FPATH
```

Comment:
```suggestion
+ FPATH=$(brew --prefix)/share/zsh/site-functions:$FPATH
```

There's no need to `export` here, I believe. I wonder if the `diff` format is clearest here or if it's just worth specifying the only specific line(s) that need added?

Reply:
Thanks, I've updated the branch based on your suggestions :)